packages/cxl-lumo-styles/scss/icons.scss

```diff
@@ -0,0 +1,26 @@
+// Unicode character map for the icons
```

Comment:
No, icons are defined at https://github.com/conversionxl/aybolit/blob/cxl-ui-0.2.0/packages/cxl-lumo-styles/src/icons.js so drop duplicate SASS implementation here.

We also load icon font https://github.com/vaadin/vaadin-lumo-styles/blob/v1.6.0/font-icons.html but don't currently have a process to add to this WOFF file, like we have for SVG-s.

Any thoughts on icon font update solution options?

Reply:
I see now you added the new icon SVG in correct location, now just gotta figure out the loader in a higher level fashion.

Reply:
This seems to be Vaadin's icon font builder routine https://github.com/vaadin/vaadin-lumo-styles/blob/v1.6.0/gulpfile.js#L166-L219 :thinking:

I wonder if adoptable, or if we need it.